Demo/Sources/PlayerView.swift

```diff
@@ -12,7 +12,10 @@ import SwiftUI
 /// Behavior: h-exp, v-exp
 struct PlayerView: View {
     let media: Media
-    @StateObject private var player = Player()
+    @StateObject private var player = Player(configuration: .init(
```

Comment:
Maybe helpful to factor out player configuration as set under _Settings_ through a method delivering the corresponding `PlayerConfiguration`?